src/actions/portfolio/index.js

```diff
@@ -24,7 +24,10 @@ export const getPortfolio = (
                 fiatCurrency,
                 coinGeckoIds
             );
-            portfolio.push(...portfolioPiece);
+
+            if (portfolioPiece) {
```

Comment:
Wouldn't it be better to enforce the fact that every portfolio handler has to return an array?

Reply:
This check is no longer required, since all the handlers return an empty array by default, after having fixed the wrong usage of `return` instead of `continue` (case of a dismissed coin). I'm going to undo this.